web/src/ui_init.js

```diff
@@ -548,7 +548,6 @@ export function initialize_everything(state_data) {
                 ],
                 {trigger},
             );
-            activity_ui.build_user_sidebar();
```

Comment:
@evykassirer FYI, since this line was introduced in 231aa098cb19cec028a189dded01ea9191021e40. I agree I don't see any function it can have had running here.

Reply:
Ah, I probably just didn't realize it was also being called in `message_view.show`. There are a few early returns in `show` but I _think_ none of those are cases we'd need to be concerned about, though I didn't look that closely.